e2etest/testcases.json

```diff
@@ -79,10 +79,18 @@
 		"case_name": "prometheus_static",
 		"platforms": ["EC2", "ECS", "EKS", "SOAKING"]
 	},
+	{
+		"case_name": "prometheus_static_adot_operator",
+		"platforms": ["EKS"]
+	},
 	{
 		"case_name": "prometheus_sd",
 		"platforms": ["EKS"]
 	},
+	{
+		"case_name": "prometheus_sd_adot_operator",
```

Comment:
can you please also add `otlp_trace` test case in case the changes are compatible for otlp cases.